### PR TITLE
Revision

### PR DESCRIPTION
**Pull request by dvnstrcklnd**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,56 @@
...
+
+* Write code for other people. What will happen when (not if) you rotate off this
+project and someone else has to maintain your code?
+* Make the code easy to deploy anywhere. Assume the person grading this has no more
+than 5 minutes to get your code running.
+* Embrace constraints, but assume they are not static. Use design patterns that 
+make your code easy to change. 
+
+In particular, we would like to see:
+1. A README that explains to a **non-programmer** what the software does and
```
Comment (bclipp) [inline README.md:19]: Its a little wordy.

File: data_engineer/README.md [hunk 2]
```diff
@@ -1,22 +1,17 @@
...
-* Un-encrypt the project direction file  ```interview_problem.md.encrypted```. The file has been encrypted using the AES-256-CBC algorithm with openssl.  
+From these files, generate an output file in json listing:
 
-* Follow the directions interview_problem.md
+1. Each student
+2. The teacher the student has
+3. he class ID the student is scheduled for
 
-### Here are some resources on openssl   
-https://wiki.openssl.org/index.php/Enc
```
Comment (bclipp) [inline data_engineer/README.md:15]: My preference would be to not have remove the requirement to decrypt the directions. It solves two issues.
1. A basic task of following the copy and past directions.
2. Having recruiting exercises free out in the open isn't the smartest strategy.

Comment (dvnstrcklnd) [reply data_engineer/README.md:15]: These aren't unreasonable goals, but we're looking to hire substantially higher in the food chain than "can copy and paste commands." We run the risk of losing good people if we ask them to do do annoyingly basic stuff. 

If we want to keep the exercises secret, there are other ways to do that.

File: README.md (hunk 1 above)
Comment (dvnstrcklnd) [reply README.md:19]: I'll give it another pass, but we want to err on the side of clarity about what were evaluating them on. This is as much about what we value as it is about their technical skill.

File: data_engineer/README.md (hunk 2 above)
Comment (bclipp) [reply data_engineer/README.md:15]: I don't think I feel comfortable being on the PR, since I am not involved in the hiring process at this time.

**Pull request merged**